src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue29325.cs

```diff
@@ -0,0 +1,20 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue29325 : _IssuesUITest
+{
+	public Issue29325(TestDevice device) : base(device) { }
+
+	public override string Issue => "Button Shadow Color Transparency Not Applied Correctly";
+
+	[Test]
+	[Category(UITestCategories.Button)]
+	public void ShouldUpdateButtonShadowWithTransparentColor()
+	{
+		App.WaitForElement("Button");
+		VerifyScreenshot();
```

Comment:
Pending snapshots already available in the latest build.
![image](https://github.com/user-attachments/assets/03a73b56-c93a-4bc3-a762-ae83fb44c00a)

Could you commit the images?